lib/samples/find_route/find_route_sample.dart

```diff
@@ -25,29 +25,34 @@ class FindRouteSample extends StatefulWidget {
 }
 
 class _FindRouteSampleState extends State<FindRouteSample> {
+  // the map view controller.
   final _mapViewController = ArcGISMapView.createController();
+  // the graphics overlay for the stops.
   final _stopsGraphicsOverlay = GraphicsOverlay();
+  // the graphics overlay for the route.
   final _routeGraphicsOverlay = GraphicsOverlay();
-  final _routeLineSymbol = SimpleLineSymbol(
-      style: SimpleLineSymbolStyle.solid, color: Colors.blue, width: 5.0);
-  final _routeStartSymbol = SimpleMarkerSymbol(
-      style: SimpleMarkerSymbolStyle.circle, color: Colors.green, size: 8.0);
-  final _routeEndSymbol = SimpleMarkerSymbol(
-      style: SimpleMarkerSymbolStyle.circle, color: Colors.red, size: 8.0);
-  final List<Stop> _stops = [];
-  var _isRouteGenerated = false;
-  var _isReady = false;
-  List<DirectionManeuver> _directions = [];
+  // the stops for the route.
+  final _stops = <Stop>[];
+  // whether the route has been generated.
+  var _routeGenerated = false;
+  // a flag for when the map view is ready used to control availability of parts of the UI.
+  var _ready = false;
+  // the directions for the route.
+  var _directions = <DirectionManeuver>[];
+  // the parameters for the route.
   late final RouteParameters _routeParameters;
+  // the route task.
   final _routeTask = RouteTask.withUrl(Uri.parse(
       'https://sampleserver6.arcgisonline.com/arcgis/rest/services/NetworkAnalysis/SanDiego/NAServer/Route'));
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       body: SafeArea(
+        // create a column with buttons for generating the route and showing the directions.
```

Comment:
Let's set the `SafeArea` to only operate at the bottom, so that the map view itself flows under the top notches etc. but we keep the buttons at the bottom in the `SafeArea`.

You can do that like:
```
SafeArea(
        top: false,
```